client/src/modules/cohort/index.jsx

```diff
@@ -98,17 +74,14 @@ class Cohort extends PureComponent {
 
       updateCohort(cohortId, dataToUpdate);
     }
-    this.hideUpdateForm();
-  };
-
-  showDialog = () => this.setState({ isDialogVisible: true });
 
-  hideDialog = () => this.setState({ isDialogVisible: false });
+    this.handleDialogContext(null)();
```

Comment:
create method `hideDialog` which will have exactly this line and use it everywhere